packages/web-ui/src/components/TextField/TextField.stories.tsx

```diff
@@ -51,5 +29,33 @@ export const Workshop: Story = {
     helperText: 'Helper text',
     placeholder: 'Placeholder',
     id: 'web-ui-textfield-example',
+    removeGutterBottom: false,
```

Comment:
I would have the other way. The default should be without the gutter and the option should be to include the gutter IMHO

Reply:
yeah, I agree, however it's only this way because I feel that removing the gutter could cause unexpected layout shift where the textfield is already in use, and should probably be a breaking change. This prop is a temporary override. We'll change this in the new `Input` component.